docs/guides/55-performance/cluster-key.md

```diff
@@ -26,7 +26,7 @@ If your primary queries involve retrieving cities based on their temperature, se
 
 ![Alt text](@site/docs/public/img/sql/clustered.png)
 
-Rows are sorted based on the Age column in each block (file). However, there can be overlapping age ranges between blocks. If a query falls precisely within the overlapping range of blocks, it requires reading multiple blocks. The number of blocks involved in this situation is referred to as the "Depth." Therefore, the smaller the depth, the better. This implies that having fewer relevant blocks to read during queries enhances query performance.
+Rows are sorted based on the Temperature column in each block (file). However, there can be overlapping age ranges between blocks. If a query falls precisely within the overlapping range of blocks, it requires reading multiple blocks. The number of blocks involved in this situation is referred to as the "Depth." Therefore, the smaller the depth, the better. This implies that having fewer relevant blocks to read during queries enhances query performance.
```

Comment:
`temperature`?

Reply:
That's from the example at the beginning of the this section: https://docs.databend.com/guides/performance/cluster-key#how-cluster-key-works